src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160420_workspace_record_version.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="thibault" id="workspace_record_version">
+        <addColumn tableName="WORKSPACE">
+            <column name="record_version" type="BIGINT">
+                <constraints nullable="false"/>
```

Comment:
when the db is empty, this will work. When there are already rows and you are adding a column you need to tell the db what to put in the column (this is true for oracle anyway)